shortlister/controller.py

```diff
@@ -8,7 +8,7 @@ class Controller:
     def __init__(self,path):
         self.path = path
         self.shortlist = load_shortlist(path)
-        self.current_applicant = None
+        self.applicant_index = int
```

Comment:
I don't think you intended this variable to be a Python `type`!

```suggestion
        self.applicant_index: int = 0
```